includes/validate.php

```diff
@@ -96,6 +92,12 @@ function edac_save_post( $post_ID, $post, $update ) {
 		}
 	}
 
+	// Post in, or going to, trash.
+	if ( 'trash' === $post->post_status ) {
+		Purge_Post_Data::delete_post( $post_ID );
```

Comment:
```suggestion
		// Gutenberg does not fire the `wp_trash_post` action when moving posts to the
		// trash. Instead it uses `rest_delete_{$post_type}` which passes a different shape
		// so instead of hooking in there for every post type supported the data gets
		// purged here instead which produces the same result.
		Purge_Post_Data::delete_post( $post_ID );
```